src/components/Dialog/types.ts

```diff
@@ -22,6 +22,7 @@ export type DialogControlRefProps = {
 export type DialogControlProps = DialogControlRefProps & {
   id: string
   ref: React.RefObject<DialogControlRefProps>
+  isOpen: boolean
```

Comment:
Added so that we can control the Radix dropdowns using the same `control` interface